pipeline/review/main.go

```diff
@@ -165,5 +166,19 @@ func newQualifiedTableName(s string) (*bigqueryio.QualifiedTableName, error) {
 	if strings.TrimSpace(project) == "" || strings.TrimSpace(dataset) == "" || strings.TrimSpace(table) == "" {
 		return nil, fmt.Errorf("table name has empty components: %s", s)
 	}
+	var merr error
```

Comment:
There are some existing validations above - can they be combined here?

Reply:
Hmm, I think I'll do a bit of a rewrite of this.

It doesn't handle invalid inputs with more than two `.` 

I'll make some changes and see how it looks

Reply:
Committed a change, PTAL